### ISSUE TITLE
Change e-mail subjects for Changesets to ones that mention the changeset # instead of user names.

### ISSUE THREAD

**Issue by rickmastfan67** [openstreetmap/openstreetmap-website#1050]
Currently, when there is a comment on a changeset your either watching are participating in, whenever more than one person comments in it, the website sends out an e-mail with that person's name in it.  If 4-5 different people make a comment, it makes it extremely hard to follow the comments on the e-mail end since some e-mail sites merge all e-mails with the same subject (like gmail), breaking any possibility to follow the conversation, especially if the same user commented on another totally different changeset that you were subscribed to.

For example, the following changeset (http://www.openstreetmap.org/changeset/33669446) is very active right now and I've gotten several e-mails with 5 different subjects since that's how many people have responded in it.

So, what I'm suggesting is to change the way the 'subjects' are made for these e-mails.  Instead of saying a specific user has commented in it, I would suggest it mention the changset # instead.

Example, instead of:
[OpenStreetMap] < user name here > has commented on a changeset you are interested in
How about something like this:
[OpenStreetMap] Changeset #xxxxxx: A new comment has been made.

This way, instead of having 5 different e-mail subject clogging up your Inbox, there would just be one subject. That way, an e-mail provider can then merge all the e-mails together and a proper conversation can then be read instead of having to guess what e-mail came in which order.

**Comment by polarbearing** [comment]
The changeset number should be included in the subject. It might not be necessary to remove the user, so I would propose:

```[OpenStreetMap] USER has commented on changeset NUMBER you are interested in```
or
```[OpenStreetMap] Changeset NUMBER you are interested in has been commented by USER```

This would allow to filter mails by either user name or number as appropriate.
